substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/resources/NativeImageResourceFileSystemProvider.java

```diff
@@ -70,8 +71,8 @@ private Path uriToPath(URI uri) {
         }
 
         try {
-            // Syntax for URI resource is resource:file:resources!/{entry}
```

Comment:
So that comment was wrong all along, right?

Reply:
Unfortunately yes, I changed implementation slightly during the review period but forgot to update the comment accordingly.